docs/_tutorials/getting_started.md

```diff
@@ -176,7 +176,8 @@ const app = new App({
   token: process.env.SLACK_BOT_TOKEN,
   signingSecret: process.env.SLACK_SIGNING_SECRET,
   socketMode: true,
-  appToken: process.env.SLACK_APP_TOKEN
+  appToken: process.env.SLACK_APP_TOKEN,
+  port: process.env.PORT || 3000 // socket mode doesn't listen on a port, but in case you want your app to respond to OAuth, you still need to listen on some port!
```

Comment:
How does this comment look in the live document? I could be totally wrong, but it seems like it will required a far scroll right to read. If so, is there a way to truncate it while keeping the same meaning?

Reply:
Here's a little screenshot from the locally running server on my Chrome:

![Screen Shot 2021-11-08 at 8 12 47 PM](https://user-images.githubusercontent.com/52645/140843465-bb4fd02e-798b-4575-a359-e709d92d9a0a.png)

Reply:
Let me know what you think!

Reply:
Ah, on both points I entirely forgot this was strictly limited to the Getting Started layout. LGTM :shipit: